raiden-ts/src/path/actions.ts

```diff
@@ -14,7 +14,10 @@ type ServiceId = {
   serviceAddress: Address;
 };
 
-export const pathFind = createStandardAction('pathFind')<{ paths?: Paths; pfs?: PFS }, PathId>();
+export const pathFind = createStandardAction('pathFind')<
```

Comment:
`typesafe-actions` v4 seem to have a new way of creating standard actions. Do you see any benefit of migrating to the new style when we create new actions or touch old ones?

Reply:
Hmm, which new way? From [what I can see](https://github.com/piotrwitek/typesafe-actions#2-fsa-compliant-actions), this is still how it's done. Anyway,  I'd say we shouldn't touch this now, and consider #591 when we need to migrate to anything else (probably a custom implementation based on typesafe-actions and io-ts by then)

Reply:
We had a dependency PR https://github.com/raiden-network/light-client/pull/474#issuecomment-549709382 that tried to bump the version to 5.x which deprecates the way actions are created in 4.x (like we do currently). The "old" way still works if we use `createStandardAction` from `deprecated`

Reply:
Hmm, right.. didn't see the changes in new release, we may move forward then to the serializable approach. Thank you for pointing this out.